dandischema/metadata.py

```diff
@@ -181,9 +181,21 @@ def validate(
             klass = getattr(models, schema_key)
             schema = klass.schema()
         else:
+            schema_map = {
+                "Dandiset": "dandiset",
+                "PublishedDandiset": "published-dandiset",
+                "Asset": "asset",
+                "PublishedAsset": "published-asset",
+            }
```

Comment:
Later (away from laptop ATM) we should centralize (the same dict should be used there) with with https://github.com/dandi/dandischema/blob/master/dandischema/metadata.py#L90

Reply:
FWIW -- pushed quick RF to "centralize" in 44845973ed0e90a117c7ef427439655f8dd4632f